chain/transaction_test.go

```diff
@@ -74,6 +77,73 @@ func unmarshalAction2(p *codec.Packer) (chain.Action, error) {
 	return &action, err
 }
 
+func TestJSONMarshalUnmarshal(t *testing.T) {
+	require := require.New(t)
+
+	txdata := chain.TransactionData{
```

Comment:
nit: camel case to `txData`

separate from this PR, but we should be consistent with the variable name throughout this file as well